core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -99,6 +104,56 @@ public S3RestServiceHandler(@Context ServletContext context) {
         (FileSystem) context.getAttribute(ProxyWebServer.FILE_SYSTEM_SERVLET_RESOURCE_KEY);
   }
 
+  private static String getUserFromAuthorization(String authorization) {
```

Comment:
Can we make this not `private`, and then add a unittest for this method?

Reply:
There is a general lack of unit tests for most methods here.. That is partially because I don't have a good understanding of the full API spec quite yet and I plan to add them later as I implement more and have a better understanding of the spec.

Reply:
Ok, as long as we don't forget to add the coverage soon.